src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,13 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String getPrintableString(Printable... printables){
+    	String printString="";
+    	for(Printable printable: printables){
```

Comment:
Good use of "for each" to make the code cleaner. Note the coding style though, there should be a space after "for".